scripts/mkcloud

```diff
@@ -540,24 +540,84 @@ function setupadmin()
     fi
 }
 
+function shutdown_node() {
+    virsh shutdown $1
+}
```

Comment:
I'll revert those calls and remove the function. That will make the `virsh start` calls consistent across the code.